backend/infrahub/workers/utils.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+import importlib
+from typing import TYPE_CHECKING, Any
+
+from prefect import Flow
+
+if TYPE_CHECKING:
+    from infrahub.services import InfrahubServices
+
+# TODO solve this circular issue. we should have no ref to WorkflowLocalExec within InfrahubServices
+
+
+def inject_service_parameter(service: InfrahubServices, parameters: dict[str, Any]) -> None:
```

Comment:
I would suggest we call it `inject_parameters` instead, We'll need to inject more than just `service` so let's make sure the name of the functions (and the code in general) are not too specific
The next one it would be good to inject is the Infrahub client